### ISSUE TITLE
Add meaningful tooltip on the calendar selector

### ISSUE THREAD

**Issue by mattab** [piwik/piwik#9557]
Goal of this issue is to improve accessibility of the Calendar selector in Piwik, by making it easier to discover.

When hovering on the calendar [in this report:](http://demo.piwik.org/index.php?module=CoreHome&action=index&date=yesterday&period=week&idSite=7#?module=Live&action=indexVisitorLog&date=yesterday&period=week&idSite=7)

 * Current tooltip says `Choose date`
 * Instead we expect a more useful tooltip such as: `Choose date, currently selected date is [week January 11 – 17, 2016]`

A more meaningful tooltip will help users know the currently selected date even if they cannot actually read the calendar. This is especially useful along with #9550

**Comment by mattab** [comment]
Done in #9559
